Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -0,0 +1,215 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// Controller responsible for displaying the channel messages.
+@available(iOSApplicationExtension, unavailable)
+open class ChatChannelVC: _ViewController, ThemeProvider {
```

Comment:
I would recommend put ThemeProvider conformance to `open extension` to make it more readable.
https://github.com/raywenderlich/swift-style-guide#protocol-conformance

Reply:
Theme Provider doesn't require implementations. so the extension would be `extensions ChatChannelVC: ThemeProvider {}` which is way less clear.

Reply:
Also as idea: we could put a `Note:` comments for empty extension